omero/autogen_docs

```diff
@@ -51,6 +51,7 @@ for f in \
     omero-* \
     settings.env \
     setup_* \
+    docker_virtualenv*\
```

Comment:
I think the idea is that these should be specific to the testing, and shouldn't be needed in the docs.